tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,116 +1,86 @@
 const expect = require('chai').expect
-var { sequentialPromise, wait } = require('../lib/utils');
-const peglib = require('peglib');
-const bitcoin = peglib.bitcoin;
-const rsk = peglib.rsk;
-const pegUtils = peglib.pegUtils;
-const pegAssertions = require('../lib/assertions/2wp');
-const whitelistingAssertionsLegacy = require('../lib/assertions/whitelisting-legacy');
-const CustomError = require('../lib/CustomError');
 const rskUtils = require('../lib/rsk-utils');
 const { getRskTransactionHelpers } = require('../lib/rsk-tx-helper-provider');
 const { getBtcClient } = require('../lib/btc-client-provider');
+const { sendPegin, ensurePeginIsRegistered, sendTxToBridge, MINIMUM_PEGIN_VALUE_IN_BTC, MINIMUM_PEGOUT_VALUE_IN_RBTC } = require('../lib/2wp-utils');
+const { getBridge, getLatestActiveForkName } = require('../lib/precompiled-abi-forks-util');
+const { getDerivedRSKAddressInformation } = require('@rsksmart/btc-rsk-derivation');
+const btcEthUnitConverter = require('btc-eth-unit-converter');
+const whitelistingAssertions = require('../lib/assertions/whitelisting');
 
 describe('Transfer BTC to RBTC before papyrus200', function() {
 
-  var federationAddress;
-  var btcClient;
-  var rskClient;
-  var pegClient;
-  var test;
-  let whitelistTestLegacy;
   let rskTxHelpers;
   let btcTxHelper;
+  let rskTxHelper;
 
-  const NETWORK = bitcoin.networks.testnet;
+  
+  const PEGIN_VALUE_IN_SATOSHI = btcEthUnitConverter.btcToSatoshis(3 * MINIMUM_PEGIN_VALUE_IN_BTC);
+  const MINIMUM_PEGOUT_VALUE_IN_WEIS = btcEthUnitConverter.btcToWeis(MINIMUM_PEGOUT_VALUE_IN_RBTC);
+  const PEGIN_VALUE_IN_BTC = 3 * MINIMUM_PEGIN_VALUE_IN_BTC;
+  const PEGIN_VALUE_IN_WEISS = btcEthUnitConverter.btcToWeis(PEGIN_VALUE_IN_BTC)
+  const PEGOUT_VALUE_IN_RBTC = PEGIN_VALUE_IN_BTC / 3
+  const RSK_TX_FEE_IN_RBTC = 0.001
```

Comment:
```suggestion
  const PEGIN_VALUE_IN_BTC = 3 * MINIMUM_PEGIN_VALUE_IN_BTC;
  const PEGOUT_VALUE_IN_RBTC = PEGIN_VALUE_IN_BTC / 3;
  const PEGOUT_VALUE_IN_WEIS = btcEthUnitConverter.btcToWeis(PEGOUT_VALUE_IN_RBTC);
  const RSK_TX_FEE_IN_RBTC = 0.001
```
Tried to simplify it a bit there, let me know what you think. We are not really dealing with minimum values here, just doing a peg-in and then 2 peg-outs right? So keeeping that in mind I think we can simplify it a lot